database/index.js

```diff
@@ -1,6 +1,7 @@
 //dev2
 
 const mongoose = require('mongoose');
+//const uri = 'mongodb://mongo:27017/reviewsDB';
```

Comment:
is this one commented out? Isn't this the one you need for docker?

Reply:
you rights, i had to wrong one commented out